### DISCUSSION TITLE
Management PipeCD v0.45.4 features

### DISCUSSION THREAD

**Discussion by khanhtc1202** [pipe-cd/pipecd#4574] [Ideas]
About PipeCD release cycle: [Release docs](https://github.com/pipe-cd/pipecd/blob/master/RELEASES.md#release-cycle)
v0.45.4 project board: [v0.45.4](https://github.com/orgs/pipe-cd/projects/6)

Any other ideas/requests are more than welcome. Feel free to add as comments under this discussion 🙏

**Comment by t-kikuc** [comment]
@khanhtc1202 
Could we close this discussion? cuz v0.46.0 is released.
